mlflow/tensorflow.py

```diff
@@ -694,12 +694,30 @@ def _setup_callbacks(lst):
+    |                                            | stopped_epoch, restored_epoch, restore_best_weight, etc     | For example, min_delta, patience, baseline, restore_best_weights,etc               |               |                                                                                                                                                  |
+    +--------------------------------------------+-------------------------------------------------------------+------------------------------------------------------------------------------------+---------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+    | ``tf.estimator``                           | TensorBoard metrics. For example average_loss, loss etc     | steps, max_steps                                                                   | --            | `MLflow Model <https://mlflow.org/docs/latest/models.html>`_ (TF saved model) on call to ``tf.estimator.export_saved_model``                     |
+    +--------------------------------------------+-------------------------------------------------------------+------------------------------------------------------------------------------------+---------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+    | TensorFlow Core                            | All ``tf.summary.scalar`` calls                             | --                                                                                 | --            | --                                                                                                                                               |
+    +--------------------------------------------+-------------------------------------------------------------+------------------------------------------------------------------------------------+---------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+
+    Note that autologging for ``tf.keras`` is handled by :py:func:`mlflow.tensorflow.autolog`, not :py:func:`mlflow.keras.autolog`.
+    Whether you are using TensorFlow 1.x or 2.x, the respective metrics associated with ``tf.estimator`` and ``tf.keras.callbacks.earlyStopping`` are automatically logged.
+    As an example, try running the `MLflow examples/tensorflow <https://github.com/mlflow/mlflow/tree/master/examples/tensorflow>`_.
+
+    Refer to the autologging tracking documentation for
```

Comment:
Probably don't need this statement since these docs explain the workflow already.

Reply:
Agreed, is a bit redundant.

Reply:
Would it make more sense to point to the TF 1.x and TF 2.x examples specifically when we mention the examples at the beginning of this doc?